crates/virtio-queue/src/iterator.rs

```diff
@@ -102,7 +102,7 @@ pub struct AvailIter<'b, M> {
 impl<'b, M> AvailIter<'b, M>
 where
     M: Deref,
-    M::Target: GuestMemory + Sized,
+    M::Target: GuestMemory,
```

Comment:
Can you add a description about this in the commit message as well? It will help with writing the changelog.

Reply:
Hmm, I think it's better to relax the trait bounds for the other methods as well first via a separate PR, and add that to the changelog. Created #152 to track this.

Reply:
Is this change required in this commit? If not, I would postpone it when fixing #152, if yes, I would still leave some sort of explanation about it in the commit message or at least give a pointer to #152.